src/main/java/io/oneko/configuration/SchedulerConfig.java

```diff
@@ -1,30 +1,23 @@
 package io.oneko.configuration;
 
 import java.time.Clock;
-import java.util.concurrent.Executor;
-import java.util.concurrent.Executors;
 
+import org.springframework.boot.task.TaskSchedulerBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.scheduling.TaskScheduler;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
 
 @Configuration
 public class SchedulerConfig {
-	@Bean
-	public TaskScheduler taskScheduler() {
-		final ThreadPoolTaskScheduler scheduler = new ThreadPoolTaskScheduler();
-		scheduler.setPoolSize(10);
-		return scheduler;
-	}
 
-	@Bean()
-	public Executor taskExecutor() {
-		return Executors.newScheduledThreadPool(10);
+	@Bean
+	public ThreadPoolTaskScheduler taskScheduler(TaskSchedulerBuilder builder) {
```

Comment:
It's just an easy way to configure the threadpoolsize. And since we inject the `TaskSchedulerBuilder`, the pool size also applies to the ThreadPoolTaskScheduler

Reply:
Thanks!